### PR TITLE
Prefer the last //# sourceMappingURL in a file instead of the first

### PR DESCRIPTION
**Pull request by papandreou**
## Description

It seems like some Webpack setups (including mine) end up outputting several `//# sourceMappingURL=...` entries when both transpilation and bundling is in the picture.

In general it seems like the safest bet to go for the last one, as most tools tend to append the comment to the end of the file.
Thus the last one is most likely to pertain to the last operation that happened, be it transpilation, bundling, or uglification :)

## Motivation and Context

In my Webpack setup, stacktrace-gps tries to fetch a non-existent source map because my bundle contains multiple `//# sourceMappingURL=...` entries. This worked fine before https://github.com/stacktracejs/stacktrace-gps/issues/33 was merged, but now stacktrace-gps prefers the first `//# sourceMappingURL=...` when the file contains more than one.

## How Has This Been Tested?

In my own application that uses Webpack and via a new unit test.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] `node_modules/.bin/jscs -c .jscsrc stacktrace-gps.js` passes without errors
- [x] `npm test` passes without errors
- [x] I have read the [contribution guidelines](CONTRIBUTING.md)
- [ ] I have updated the documentation accordingly
- [x] I have added tests to cover my changes

### CODE REVIEW

**Comment by cartant** [comment]
Can any guidance be offered as to if/when this PR will be merged? With tools like `ng-cli` the Webpack configuration is managed by other tools and if said tools effect bundles with multiple `sourceMappingURL` entries, there's not a lot that can be done to address the situation by the tools' users.

File: stacktrace-gps.js [hunk 1]
```diff
@@ -321,12 +326,13 @@
                         sourceMappingURL = defaultSourceRoot + sourceMappingURL;
                     }
 
-                    return this._getSourceMapConsumer(sourceMappingURL, defaultSourceRoot).then(function(sourceMapConsumer) {
-                        return _extractLocationInfoFromSourceMapSource(stackframe, sourceMapConsumer, sourceCache)
-                            .then(resolve)['catch'](function() {
-                            resolve(stackframe);
+                    return this._getSourceMapConsumer(sourceMappingURL, defaultSourceRoot)
```
Comment (eriwen) [inline stacktrace-gps.js]: Though this is unrelated to the change, I appreciate the cleanup here. 👍

File: spec/stacktrace-gps-spec.js [hunk 2]
```diff
@@ -327,6 +327,21 @@ describe('StackTraceGPS', function() {
             }
         });
 
+        it('ignores all but the last sourceMappingURL in the file', function (done) {
```
Comment (eriwen) [inline spec/stacktrace-gps-spec.js]: Yay tests!

File: stacktrace-gps.js (hunk 1 above)
Comment (papandreou) [inline stacktrace-gps.js]: I included that that in [a separate commit](https://github.com/stacktracejs/stacktrace-gps/pull/53/commits/331754fbd147b4600ecc4da5744ae172e792459f) so that I could tick the "`node_modules/.bin/jscs -c .jscsrc stacktrace-gps.js` passes without errors" box ;)
